globus_cli/services/transfer.py

```diff
@@ -319,18 +319,18 @@ def check_completed():
         )
         if completed:
             if heartbeat:
-                safeprint("", write_to_stderr=True)
+                click.echo("", err=True)
             # meowing tasks wake up!
             if meow:
-                safeprint(
+                click.echo(
                     r"""
                   _..
   /}_{\           /.-'
  ( a a )-.___...-'/
  ==._.==         ;
       \ i _..._ /,
       {_;/   {_//""",
```

Comment:
😻

Reply:
Hands down the best part of this product to date. I'm invested in testing it after each release.